diff --git a/node_modules/vscode/vscode/src/vs/editor/standalone/browser/standaloneServices.js b/node_modules/vscode/vscode/src/vs/editor/standalone/browser/standaloneServices.js
index e344741..4a5a2a2 100644
--- a/node_modules/vscode/vscode/src/vs/editor/standalone/browser/standaloneServices.js
+++ b/node_modules/vscode/vscode/src/vs/editor/standalone/browser/standaloneServices.js
@@ -642,7 +642,7 @@ let StandaloneResourcePropertiesService = class StandaloneResourcePropertiesServ
         if (eol && typeof eol === 'string' && eol !== 'auto') {
             return eol;
         }
-        return (isLinux || isMacintosh) ? '\n' : '\r\n';
+        return (isLinux || isMacintosh) ? '\n' : '\n';
     }
 };
 StandaloneResourcePropertiesService = ( __decorate([
